src/components/SearchBar.js

```diff
@@ -42,10 +45,25 @@ class SearchBar extends Component {
 
     render() {
         return (
-            <form onSubmit={this.handleSubmit}>
-                <input type="text" value={this.state.value} onChange={this.handleChange} />
-                <input type="submit" value="Submit" />
-            </form>
+            <div>
+                <MuiThemeProvider>
+                    {/*TODO: Make AutoComplete component do its job! (Hint: lines 52 & 12!)*/}
+                    <AutoComplete 
```

Comment:
Very interesting, do you anticipate using the tags/categories SFWAR has created as part of their taxonomy?